coffeecard/CoffeeCard.Tests.Unit/Services/AccountServiceTest.cs

```diff
@@ -142,5 +148,155 @@ await using (var context = new CoffeeCardContext(builder.Options, databaseSettin
+        {
+            // Arrange
+            var builder = new DbContextOptionsBuilder<CoffeeCardContext>()
+                .UseInMemoryDatabase(nameof(LoginTimeoutsAfterFiveFailedLogins));
+
+            var databaseSettings = new DatabaseSettings()
+            {
+                SchemaName = "test"
+            };
+            var environmentSettings = new EnvironmentSettings(){DeploymentUrl = "test", EnvironmentType = EnvironmentType.Test, MinAppVersion = "2.1.0"};
+            var identitySettings = new IdentitySettings(){AdminToken = "test",MaximumLoginAttempts = 5, TimeOut = new TimeSpan(1,0,0), TokenKey = "token"};
+
+            var userTokens = new List<Token>() {};
+            var user = new User {Id = 1, Name = "test", Tokens = userTokens, Email = "test@email.dk", Programme = new Programme(), Password = "test", IsVerified = true};
+
+            var wrongPass = "wrongPassword";
+            var hasher = new Mock<IHashService>();
+            hasher.Setup(h => h.Hash(wrongPass)).Returns(wrongPass);
+
+            var httpContextAccessor = new Mock<IHttpContextAccessor>();
+            httpContextAccessor.Setup(h => h.HttpContext).Returns(new DefaultHttpContext().HttpContext);
+            
+            // Act
+            await using (var context = new CoffeeCardContext(builder.Options, databaseSettings))
+            {
+                context.Add(user);
+                context.SaveChanges();
+
+                var accountService = new AccountService(context, environmentSettings, new Mock<ITokenService>().Object,
+                    new Mock<IEmailService>().Object, hasher.Object, httpContextAccessor.Object,
+                    new LoginLimiter(), identitySettings );
+
+                //Attempts to login 5 time with the wrong credentials 
+                Assert.Throws<ApiException>(() => accountService.Login(user.Email, wrongPass, "2.1.0"));
+                Assert.Throws<ApiException>(() => accountService.Login(user.Email, wrongPass, "2.1.0"));
+                Assert.Throws<ApiException>(() => accountService.Login(user.Email, wrongPass, "2.1.0"));
+                Assert.Throws<ApiException>(() => accountService.Login(user.Email, wrongPass, "2.1.0"));
+                Assert.Throws<ApiException>(() => accountService.Login(user.Email, wrongPass, "2.1.0"));
+
+                //Attempts to login a sixth time with the same credentials and captures the exception 
+                var tooManyLoginsException = Record.Exception(() => accountService.Login(user.Email, wrongPass, "2.1.0"));
+                // Assert
+                Assert.NotNull(tooManyLoginsException);
+                Assert.IsType<ApiException>(tooManyLoginsException);
+                var tooManyLoginsApiException = (ApiException) tooManyLoginsException;
+                Assert.Equal(429,tooManyLoginsApiException.StatusCode); //Determines that the exception from the sixth attempt is due to too many logins
```

Comment:
This would be cleaner
```suggestion
var expectedException = new ApiException($"Amount of failed login attempts exceeds the allowed amount, please wait for xx minutes", 429);
Assert.Equal(tooManyLoginsException, expectedException);
```